### PR TITLE
Document the load src code from url, gist feature

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
fixes https://github.com/lfortran/lcompilers_frontend/issues/102

fixes https://github.com/lfortran/lcompilers_frontend/issues/104

### CODE REVIEW

**Comment by certik** [comment]
Thanks for adding documentation. I think this does not fix https://github.com/lfortran/lcompilers_frontend/issues/102, right? To fix it, we have to document that github will delay this by N minutes, or whatever exactly happens.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,40 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
...
+##### Example URL:
+```
+https://dev.lfortran.org/?code=program%20hello%0A%20%20!%20This%20is%20a%20comment%20line%3B%20it%20is%20ignored%20by%20the%20compiler%0A%20%20print%20*%2C%20%27Hello%2C%20World!%27%0Aend%20program%20hello%0A
+```
+
+#### Loading Source Code from GitHub Gist
+
+- Alternatively, users can load source code from a GitHub Gist by providing the Gist URL in the format: `https://dev.lfortran.org/?gist=<GitHub username>/<Gist ID>`.
+- The compiler will fetch the raw content from the specified GitHub Gist.
+- Note that GitHub Gist updates the raw content approximately every 5 minutes, so any changes made to the Gist may take about 5 minutes to reflect when loaded.
```
Comment (Shaikh-Ubaid) [inline README.md:47]: >Thanks for adding documentation. I think this does not fix [#102](https://github.com/lfortran/lcompilers_frontend/issues/102), right? To fix it, we have to document that github will delay this by N minutes, or whatever exactly happens.

Yes, I detailed it here.

Comment (Shaikh-Ubaid) [reply README.md]: I updated/improved the wording.

**Review by certik — APPROVED** [review]
Looks great!

**Pull request merged**